### PR TITLE
2304 rollback

### PR DESCRIPTION
**Pull request by tinybike**
[Clubhouse Story](https://app.clubhouse.io/augur/story/2304)

## Submitter checklist
- [x] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [x] Steps to reproduce
  - [ ] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

File: src/modules/markets/actions/load-markets-info-if-not-loaded.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { loadMarketsInfo } from 'modules/markets/actions/load-markets-info'
+import { isMarketLoaded } from 'modules/market/helpers/is-market-loaded'
+import logError from 'utils/log-error'
+
+export const loadMarketsInfoIfNotLoaded = (marketIds, callback = logError) => (dispatch, getState) => {
+  const { marketsData } = getState()
+  const marketIdsToLoad = marketIds.filter(marketId => isMarketLoaded(marketId, marketsData))
+  console.log('marketIdsToLoad:', marketIdsToLoad)
```
Comment (bthaile) [inline src/modules/markets/actions/load-markets-info-if-not-loaded.js]: const marketIdsToLoad = marketIds.filter(marketId => isMarketLoaded(marketId, marketsData))

should it be `!isMarketLoaded` not market loaded? I made this change and now getting market data loaded. wanted to check with you.

Comment (tinybike) [reply src/modules/markets/actions/load-markets-info-if-not-loaded.js]: Good catch, yes it is backwards here!

**Comment by tinybike** [comment]
@bthaile Tests are passing locally for me, and also passing in Travis.  Suspect the problem is that you're locally linked to a version of augur.js that isn't the same as the one in package.json.

**Comment by bthaile** [comment]
tests are passing and looks good.

**Pull request merged**